_test/main.tf

```diff
@@ -3,3 +3,13 @@ module "s3-bucket-with-cloudfront" {
 
   bucket_name = ""
```

Comment:
that's an odd test using a bucket name which is obviously invalid. I'd recommend to use bucket name starting with `example-`